src/Refact/Internal.hs

```diff
@@ -358,7 +356,7 @@ droppedComments rs orig_m m = not (all (`Set.member` current_comments) orig_comm
     isComment :: GHC.EpaComment -> Bool
     isComment _ = True
     orig_comments = all_comments mcs
-    current_comments = Set.fromList $ all_comments m
+    current_comments = all_comments m
```

Comment:
Why dropping `Set.fromList`?

Reply:
Because `(all_comments m):: [EpaComment]` doesn't have `Eq` constraint since ghc-9.4, and I don't know any necessity to implement it manually.

https://hackage.haskell.org/package/ghc-9.4.2/docs/GHC-Parser-Annotation.html#t:EpaComment
https://hackage.haskell.org/package/ghc-9.2.4/docs/GHC-Parser-Annotation.html#t:EpaComment

Reply:
Ah I see, the `Ord` instance was removed.

Reply:
Oh yes, it's Ord😅